frb/halos.py

```diff
@@ -88,7 +89,7 @@ def frac_in_halos(zvals, Mlow, Mhigh, rmax=1.):
 
         # Setup
         #dndlM = np.array([hmfe.dndlnM(Mi, a)[0] for Mi in M])
-        dndlM = hmfe.dndlnM(M, z)
+        dndlM = M*hmfe.dndM(M, z)
```

Comment:
Ah, right.  the `dndlM` method is a fork of mine, I think.
I can't remember if I ever issued a PR.  Probably not.

But if your approach is sound we can go with that.

Reply:
I think fg_analysis branched off master. Not sure. In any case, I'll fix it later if it breaks. This works for now